tests/regression/txt-suites/clausalmods-german

```diff
@@ -7,4 +7,4 @@ wir wohnten wie ich dir gesagt habe
 *wir wohnten ich dir gesagt habe wie
 *wir wohnten ich dir gesagt habe
 *wohnten wir wie ich dir gesagt habe
-*wir wohnten wie ich dir gesagt
+*wir wohnten wie ich dir gesagt
```

Comment:
Ideally this would end in a newline, even if the current system can handle it without. Fix with something like this:

```console
$ echo >> tests/regression/txt-suites/clausalmods-german
```

Reply:
Added a newline (though I don't actually know why they are needed).

Reply:
@olzama in text files, each line, including the last one, is supposed to be terminated with `\n`, so the POSIX-compliant `read` command just ignores the final sentence (as you can see in my comment [above](https://github.com/delph-in/matrix/pull/477#issuecomment-657167692)). The result was our old regression testing system created skeletons that did not include the final sentence.

Reply:
That seems brittle! And is this (that any text file should end in \n) considered the expected style in python specifically, or?.. But anyway, I will try to remember this.

Reply:
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline :)

Reply:
Looks like you answered your own question :) But about the Python thing, my guess is that because Python is multi-platform, it also accommodates Windows conventions.